game/world/opcode_handling/handlers/player/MovementHandler.py

```diff
@@ -51,6 +52,9 @@ def handle_movement_status(world_session, socket, reader: PacketReader) -> int:
                 world_session.player_mgr.location.z = z
                 world_session.player_mgr.location.o = o
 
+                if flags & (MoveFlags.MOVEFLAG_MOVE_MASK | MoveFlags.MOVEFLAG_STRAFE_MASK):
+                    world_session.player_mgr.on_relocation()
```

Comment:
@GrenderG @Fluglow @devwar87 if move `on_relocation` to `PlayerManager.update`, we gonna call it multiple times between the movement flag is actually changed after player pressing keys (on screen below moving=1 means flag `MoveFlags.MOVEFLAG_MOVE_MASK | MoveFlags.MOVEFLAG_STRAFE_MASK` was set, handling_movement_status = `MovementHandler.handle_movement_status` invocation).

![image](https://user-images.githubusercontent.com/3616930/166233743-32bebea6-8d6f-478d-b471-ba1d2e9206d0.png)